pkgs/build-support/cc-wrapper/default.nix

```diff
@@ -343,7 +343,9 @@ stdenv.mkDerivation {
       hardening_unsupported_flags+=" stackprotector pic"
     '' + optionalString (targetPlatform.libc == "newlib") ''
       hardening_unsupported_flags+=" stackprotector fortify pie pic"
-    '' + optionalString targetPlatform.isNetBSD ''
+      '' + optionalString (targetPlatform.libc == "musl" && targetPlatform.isi686) ''
```

Comment:
is this only a problem for i686, or 32 bit in general?

Reply:
musl32 is declared as i686-unknown-linux-musl

Reply:
I got the same issue on i486 and i586 as well. This should be ```targetPlatform.isx86_32``` to cover those cases.

Reply:
Make sure spacing is consistent here as well.